[Code] Add doc for 7.3 (#42392)
elastic/kibana: docs/code/code-basic-nav.asciidoc

[role="screenshot"]
image::images/code-blame.png[]

[float]
==== Branch selector

> You can use the Branch selector to view different branches of a repo. Note that code intelligence and search index will not be available for any branch other than master branch.

Reviewer: ```suggestion
You can use the Branch selector to view different branches of a repo. Note that code intelligence and search index are not available for any branch other than the master branch.
```
